### PR TITLE
Update tooltip position and rotation in Discussion Notes panel

### LINKED ISSUE #8937: "Minimize" and "Close" Tooltip Issues in Discussion Notes
**Description:**
- [ ] The "Minimize" tooltip appears rotated and misaligned in the Discussion Notes panel.
Image

- [ ] The "Close" tooltip is partially moving out of the screen, making it hard to read.
Image

**Steps to Reproduce:**
1. Navigate to the Discussion Notes panel.
2. Hover over the "Minimize" button to see the tooltip rotation issue.
3. Hover over the "Close" button to see the tooltip moving off-screen.

**Expected Behavior:**
- The "Minimize" tooltip should display horizontally and align correctly above or below the button.
- The "Close" tooltip should remain fully visible within the screen boundaries.

**Desktop:**
- OS: Windows
- Browser: Brave
- Version: 1.71.118

### PR DESCRIPTION
**Pull request by shauryag2002**
## Proposed Changes

- Fixes #8937 
- Corrected the conditional class name for tooltip rotation.
- Ensured consistent tooltip positioning based on the `show` state.
image
image

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Enhanced visual feedback for minimizing and closing patient notes with updated icons and tooltips.
- **Bug Fixes**
	- Improved tooltip text to accurately reflect the current state of the notes section.
- **Refactor**
	- Adjusted rendering logic and state management for better user interface experience.

### CODE REVIEW

**Review by shivankacker — CHANGES_REQUESTED** [review]
I think rather than updating the tooltip class for minimize, you should update the parent element to not be rotated. The rotation should only be added to the CareIcon if at all needed, or you can just change the `icon` to `l-angle-down` conditionally.

File: src/components/Facility/PatientNotesSlideover.tsx [hunk 1]
```diff
@@ -153,15 +153,16 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
...
       >
         <CareIcon
-          icon="l-angle-up"
+          icon={show ? "l-angle-down" : "l-angle-up"}
           className="tooltip text-lg transition-all delay-150 duration-300 ease-out"
         />
-        <span className="tooltip-text tooltip-top rotate-[-180deg] text-xs">
+        <span
+          className={`tooltip-text tooltip-${show ? "bottom -translate-x-16" : "top"} text-xs`}
+        >
```
Comment (khavinshankar) [inline src/components/Facility/PatientNotesSlideover.tsx]: use cn or classNames utils to add conditional classes, string interpolation can be hard to read sometimes

File: src/components/Facility/PatientNotesSlideover.tsx [hunk 2]
```diff
@@ -173,7 +174,9 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
           icon="l-times"
           className="tooltip text-lg transition-all delay-150 duration-300 ease-out"
         />
-        <span className="tooltip-text tooltip-bottom -translate-x-11 text-xs">
+        <span
+          className={`tooltip-text tooltip-${show ? "bottom -translate-x-11" : "top"} text-xs`}
+        >
```
Comment (khavinshankar) [inline src/components/Facility/PatientNotesSlideover.tsx]: same here

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/components/Facility/PatientNotesSlideover.tsx (hunk 1 above)
Comment (shauryag2002) [reply src/components/Facility/PatientNotesSlideover.tsx]: @khavinshankar ,Done.

File: src/components/Facility/PatientNotesSlideover.tsx (hunk 2 above)
Comment (shauryag2002) [reply src/components/Facility/PatientNotesSlideover.tsx]: @khavinshankar ,Done.

**Review by shivankacker — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by bodhish** [comment]
@coderabbitai full review

**Review by rithviknishad — APPROVED** [review]

**Comment by rithviknishad** [comment]
@khavinshankar your review required

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**